client/src/Initial/LandingPageUzma.scss

```diff
@@ -0,0 +1,51 @@
+@import '../scssStyles/mixins';
+
+.landing-page-header {
+  position: relative;
+  height: 80vh;
+  min-height: 500px;
+  background-image: linear-gradient(
+    to right,
+    var(--purple-transparent) 20%,
+    var(--purple-transparent-more)
+  );
+  display: flex;
+  align-items: center;
+  img {
+    height: max(10vh, 100px);
```

Comment:
react is a little funny (at least with what I've noticed) if you're trying to change the size of images...
if this line is intended to change the size of the image, you'll need to define the `height` _and_ `width`